airflow/api_connexion/openapi/v1.yaml

```diff
@@ -376,6 +376,8 @@ paths:
       parameters:
         - $ref: '#/components/parameters/PageLimit'
         - $ref: '#/components/parameters/PageOffset'
+        - $ref: '#/components/parameters/SortBy'
+        - $ref: '#/components/parameters/OrderBy'
```

Comment:
I'm still a fan of making this a _single_ field, rather than two as we can then easily extend this to sort by multiple columns.

For example.

Current: 
- `endpoint?sort=asc&order_by=execution_date`
- `endpoint?sort=desc&order_by=execution_date`

Proposed:
- `endpoint?order_by=execution_date`
- `endpoint?order_by=-execution_date`

This then allows future to easily support (one or both of thse)

- `endpoint?order_by=dag_id,-execution_date` (`ORDER BY dag_id, execution_date DESC`)
- `endpoint?order_by=dag_id&order_by=-execution_date` (Same, multiple params of the same name is valid and produces a list of values in Python.)

_At the very least_ if this is kept as two parameters, than `sort_direction` and `sort_column` names are clearer -- sort and order_by names are effectively synonyms so we should use clearer names.

Reply:
This would also avoid the `sort="desc", order_by=None` I mentioned below. `order_by=<field>` and `order_by=-<field>` are handled by the first branches, and the third handles a missing `order_by`. So another :+1: here.